index.adoc

```diff
@@ -1,178 +1,101 @@
 = WildFly Documentation
+:toc:
```

Comment:
This is somewhat of a personal preference, but I think `:toc: left` looks a bit better on desktop/laptop screens. It will render on top for smaller mobile devices.

Reply:
fixed, thanks